packages/maskbook/src/plugins/Wallet/formatter.test.ts

```diff
@@ -0,0 +1,18 @@
+import type { BigNumber } from 'bignumber.js'
+jest.mock('../../utils/i18n-next', () => ({
+    i18n: {
+        t: (str: string) => str,
+    },
+}))
+import { formatBalance } from './formatter'
+
+describe('formatter', () => {
+    describe('formatBalance', () => {
+        it.each<[rawValue: BigNumber.Value, decimals: number, significant: number, result: string]>([
+            ['0', 2, 2, '0'],
+            ['0.1', 2, 2, '0'],
```

Comment:
I would appreciate for case additions from anyone.

Reply:
CC @guanbinrui

Reply:
We had unit test cases before. Now it looks they have been removed. Please make your test cases run in the CI.